scss/widgets/material/treeView/_index.scss

```diff
@@ -18,7 +18,16 @@ $material-treeview-container-padding:
   $material-treeview-border-padding
   $material-treeview-container-with-border-offset;
 
-$material-treeview-search-editor-height: 33px;
+$material-treeview-search-editor-height: null !default;
```

Comment:
I think we shouldn't use the constant sizes for this purpose. Use display-flex for parent container that includes searchEditor & scrollable container. After that remove all constants that store this magic height sizes

Reply:
Your choice is a great solution. I saw it in another ticket. But it is worth considering the established principles of development in the current version of the design system. In this case, we do not use automatic height calculations in any of the themes (generic for example). The task you are talking about is refactoring. This problem must be solved when developing a new design system.

In this pull request, I tried not to make drastic changes.

If you still vote to remove the constants and rewrite the component to flex display, let me know.

@EugeniyKiyashko, @ksercs

Reply:
I think using of constants is ok at the moment

Reply:
It allows up make our code clear and remove a magic numbers and excess styles. But... well. It is advice only to chose the better way.